src/index.ts

```diff
@@ -1,49 +1,67 @@
-#!/usr/bin/env node --es-module-specifier-resolution=node
+#!/usr/bin/env node --no-warnings
 
 import chalk from 'chalk'
-import figlet from 'figlet'
 import { Command } from 'commander'
 
 import { loadProcesses } from './lib/process/index.js'
+import version from './version.js'
```

Comment:
where is `version.js` ?

Reply:
in the build

Reply:
similarly where is `./lib/process/index.js` on the ln above